### ISSUE TITLE
Using golang:alpine image for the lnd image

### ISSUE THREAD

**Issue by yashbhutwala** [SatoshiPortal/dockers#8]
I'm just wondering as to why you are not using the alpine golang image for the lnd Dockerfile here: https://github.com/SatoshiPortal/dockers/blob/master/rpi/LN/lnd/Dockerfile

**Comment by Kexkey** [comment]
Absolutely, LND dockerfile has to be updated (it's 6 months old).  It's in the TODO.  :-)
